encoding/masking-limiting-etc.md

```diff
@@ -22,14 +22,57 @@ This article will cover:
 
 Masking refers to a broad set of techniques used to merge multiple clips,
 usually one filtered clip merged with a source clip
-according to an overlay mask.
+according to an overlay mask. 
 A mask clip may contain any information
 generated from a pixel-wise operation.
+Mask clips are usually grayscale, 
+i.e. they consist of only one plane and thus contain no color information.
+In VapourSynth, such clips use the color family `GRAY`
+and one of these formats: 
+`GRAY8` (8 bits integer),
+`GRAY16` (16 bits integer),
+or `GRAYS` (single precision floating point).
+
+
 Vapoursynth includes some basic tools for manipulating masks as well:
 
 [**std.Minimum/std.Maximum**](http://www.vapoursynth.com/doc/functions/minimum_maximum.html)
 
-TODO
+The Minimum/Maximum operations replace each pixel
+with the smallest/biggest value in its 3x3 neighbourhood. 
+The 3x3 neighbourhood of a pixel
+are the 8 pixels directly adjacent to the pixel in question
+plus the pixel itself.
+To illustrate:
+![3x3 neighborhood](images/3x3.png)
+
+The Minimum/Maximum filters
+look at the 3x3 neighbourhood of each pixel in the input image
+and replace the corresponding pixel in the output image
+with the brightest (Maximum) or darkest (Minimum) pixel in that neighbourhood.
+
+Maximum generally expands/grows a mask
+because all black pixels adjacent to white edges will be turned white,
+whereas Minimum generally shrinks the mask
+because all white pixels bordering on black ones will be turned black.
+
+See the next section for usage examples.
+
+Side note:
```

Comment:
I wonder if gitbook's markdown has something special for use cases like this (e.g. an info box or similar), but I cba to look for it now.

Reply:
The new gitbook spec does lol <https://docs.gitbook.com/content-editing/rich-content#hints-and-callouts>.
The only way to do something like this would be with HTML.

Reply:
![image](https://user-images.githubusercontent.com/10972040/61573912-ecc26780-aaa5-11e9-8e86-afb07de3efb4.png)
![image](https://user-images.githubusercontent.com/10972040/61573923-22675080-aaa6-11e9-81bf-9dd5e72b6f1d.png)
![image](https://user-images.githubusercontent.com/10972040/61573928-301cd600-aaa6-11e9-9f92-b036b1a1090d.png)
Something like this, maybe?
It's responsive to the theme changing.

Reply:
Not unexpected that the new spec has it. The old might as well, but docs are kinda lost now, I guess? Maybe waybackmachiene or archive.org has something.

Your alternative looks decent enough, though.